src/components/BounceRuleDetailed/index.jsx

```diff
@@ -1,61 +1,531 @@
-          justifyContent: "flex-end",
-        }}
-      >
-        <a href="/" style={{ textDecoration: "none", color: "black" }}>
-          Log Out
-        </a>
-      </div>
-      <div
-        style={{
-          display: "flex",
-          justifyContent: "space-between",
-          margin: "1rem 0",
-        }}
-      >
-        <div
-          style={{
-            width: "20rem",
-            fontSize: "2rem",
-          }}
+    <div className="editable">
+      <i className="sg-icon sg-icon-editor-design" />
+    </div>
+    <div className="description-info">
+      <Table className="table-fixed">
+        <TableBody>
+          <TableRow>
+            <TableCell className="description-cell">
+              <strong>Description</strong>
+            </TableCell>
+            <TableCell> {currentRule.description} </TableCell>
+          </TableRow>
+        </TableBody>
+      </Table>
+    </div>
+    <div className="detail-info">
+      <Table>
+        <TableBody>
+          <TableRow>
+            <TableCell>
+              <strong>Bounce ID</strong>
+            </TableCell>
+            <TableCell> {currentRule.id} </TableCell>
+          </TableRow>
+          <TableRow>
+            <TableCell>
+              <strong>Response Code</strong>
+            </TableCell>
+            <TableCell> {currentRule.response_code} </TableCell>
+          </TableRow>
+          <TableRow>
+            <TableCell>
+              <strong>Enhanced Code</strong>
+            </TableCell>
+            <TableCell> {currentRule.enhanced_code} </TableCell>
+          </TableRow>
+        </TableBody>
+      </Table>
+      <Table>
+        <TableBody>
+          <TableRow>
+            <TableCell>
+              <strong>Regex</strong>
+            </TableCell>
+            <TableCell> {currentRule.regex} </TableCell>
+          </TableRow>
+          <TableRow>
+            <TableCell>
+              <strong>Priority</strong>
+            </TableCell>
+            <TableCell> {currentRule.priority} </TableCell>
+          </TableRow>
+          <TableRow>
+            <TableCell>
+              <strong>Bounce Action</strong>
+            </TableCell>
+            <TableCell> {currentRule.bounce_action} </TableCell>
+          </TableRow>
+        </TableBody>
+      </Table>
+    </div>
+  </div>
+);
+
+const DetailContainerEditable = ({ currentRule, onChangeRule }) => (
+  <div className="detail-container detail-container-editable card ">
+    <div className="editable">
+      <i className="sg-icon sg-icon-editor-design" />
+    </div>
+    <div className="description-info">
+      <Table className="table-fixed">
+        <TableBody>
+          <TableRow>
+            <TableCell className="description-cell">
+              <strong>Description</strong>
+            </TableCell>
+            <TableCell>
+              <TextInput
+                onChange={e => onChangeRule(e, "description")}
+                value={currentRule.description}
```

Comment:
Let's destructure everything we need at the top like const { description, id, response_code, ... } = currentRule to make it cleaner and not have to prefix everything with currentRule